plugins/out_s3/s3.c

```diff
@@ -514,10 +542,11 @@ static int cb_s3_init(struct flb_output_instance *ins,
                       struct flb_config *config, void *data)
 {
     int ret;
+    int i;
```

Comment:
Why are declaring here and not in the for-loop where we are using it?

Reply:
This is because our coding conventions.